torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -596,7 +600,8 @@ def generate_kernel_call(
                     and triton_meta.get("configs")
                     and triton_meta.get("signature")
                 ):
-                    equal_to_1 = triton_meta["configs"][0].equal_to_1
+                    if equal_to_1 is None:
+                        equal_to_1 = triton_meta["configs"][0].equal_to_1
```

Comment:
also here, need to check triton_version_uses_attrs_dict to see whether equals_to_1 is a valid thing to be checking on the config